infra/base-images/base-builder/Dockerfile

```diff
@@ -63,10 +63,11 @@ RUN export PYTHON_DEPS="\
 RUN unset CFLAGS CXXFLAGS && pip3 install -v atheris pyinstaller==4.1
 
 # Download and install the latest stable Go.
-ADD https://storage.googleapis.com/golang/getgo/installer_linux $SRC/
```

Comment:
one change here is that ADD checks the checksum of the remote file for changes (for cache invalidation). this no longer happens now, but I guess it's not a big deal because our infra (GCB) doesn't store any persistent docker cache.

Reply:
ok then fine, but will keep in mind.